Dockerfile

```
@@ -55,7 +55,7 @@ RUN \
 
 RUN \
   # install jhipster
-  cd /home/jhipster/generator-jhipster && npm ci --production && \
+  cd /home/jhipster/generator-jhipster && \
```

I think npm ci is required.
```suggestion
  cd /home/jhipster/generator-jhipster && npm ci &&\
```

---

As far as I understood calling `npm ci` beforehand doesn't really do anything. `npm install` installs the package anyways, regardless of what `npm ci` did.

Running it on my machine works.

---

lets test.

---

It did not work. I didn't properly clean my local folder so docker build copied some of the build artifacts over into the image and because of that it worked locally.

My guess is that `npm ci` is needed after all because altough `npm install` installs `tsc` it installs it in the global node_modules from where it might not be accessible by the prepare script. Not sure though.

I added `npm ci` again and it works locally, even after properly cleaning the folder before building :) #19766